config/train-causal-transformer.yaml

```diff
@@ -1,5 +1,4 @@
 defaults:
-  - train-lm
-  - model: from_config
+  - train-from-config
```

Comment:
I'm confused: I don't see a `train-from-config.yml` anywhere.  How is this working?

Reply:
Looks like I forgot to `git add` it, which I should have done with my last commit since the LSTM commit references it as well. Fixed :)